example-integration/src/main/java/nl/rabobank/gict/payments_savings/omnikassa_frontend/test/webshop/WebshopController.java

```diff
@@ -306,4 +349,19 @@ private PaymentCompletedResponse createPaymentCompletedResponse(HttpServletReque
         String signature = request.getParameter("signature");
         return newPaymentCompletedResponse(orderId, status, signature, signingKey);
     }
+
+    private RefundDetailsResponse initiateRefundTransaction(HttpServletRequest request)
+            throws RabobankSdkException, IllegalArgumentException {
+        Money money = Money.fromEuros(EUR, BigDecimal.valueOf(Double.parseDouble(request.getParameter("amount"))));
+        InitiateRefundRequest refundRequest = new InitiateRefundRequest(money, request.getParameter("description"), VatCategory.valueOf(request.getParameter("vat")));
```

Comment:
Please verify if we can also initiate a refund when the description and/or vat category field are empty (these are optional fields).